core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -240,14 +240,16 @@ package object config {
     .longConf
     .createWithDefault(4 * 1024 * 1024)
 
+  private[spark] val SECRET_REDACTION_PROPERTY = "spark.redaction.regex"
```

Comment:
Actually, instead of this you could use `SECRET_REDACTION_PATTERN.key` and `SECRET_REDACTION_PATTERN.defaultValue` in `Utils.scala`.